common/common.go

```diff
@@ -64,7 +70,11 @@ const (
 	AttributeBody                           = "body"
 )
 
-func ResourceToTags(logger Logger, resource pcommon.Resource, tags map[string]string) (tagsAgain map[string]string) {
+func ResourceToTags(
+	logger Logger,
+	resource pcommon.Resource,
+	tags map[string]string,
+) (tagsAgain map[string]string) {
```

Comment:
`gofumpt` and `golines` did not like this file. If it's too difficult to review as-is, I can remove some of the auto-formatting changes by hand.